modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/DestinationType.java

```diff
@@ -1,10 +1,24 @@
 package org.wso2.andes.kernel;
 
+import org.wso2.andes.amqp.AMQPUtils;
```

Comment:
Never use AMQP specific classes inside the andes core. We're trying to remove AMQP library dependencies from andes core.